qiskit/transpiler/passmanager.py

```diff
@@ -157,17 +169,21 @@ def _normalize_passes(passes):
                 raise TranspilerError('%s is not a pass instance' % pass_.__class__)
         return passes
 
-    def run(self, circuits, output_name=None, callback=None):
-        """Run all the passes on circuit or circuits
+    def run(
+            self,
+            circuits: Union[QuantumCircuit, List[QuantumCircuit]],
+            output_name: str = None,
+            callback: Callable = None
+    ) -> Union[QuantumCircuit, List[QuantumCircuit]]:
+        """Run all the passes on the specified `circuits`.
```

Comment:
```suggestion
        """Run all the passes on the specified ``circuits``.
```

Reply:
Sure! For the record, I was aiming to stick to the "if it is a parameter, use single quotes; if it is something else that looks like code, use double quotes", which itself comes from https://stackoverflow.com/questions/22256995/restructuredtext-in-sphinx-and-docstrings-single-vs-double-back-quotes-or-back and is not too strong. I'll change to double-quotes everywhere instead!

Reply:
Thank you 😊